frontend/routes/Wifi.jsx

```diff
@@ -10,44 +28,268 @@ export class Component extends React.Component {
 
     this.state = {
       wifi_networks: [],
-      last_refresh: '',
+      last_refresh: ''
     };
+
+
+
     setInterval(() => {
-      exec_table('nmcli dev wifi').then(result => this.setState({ wifi_networks: result }));
-      exec('date').then(result => this.setState({ last_refresh: result }));
+      exec_table('nmcli dev wifi')
+      .then(result => {
+        if(!result)
+          return;
+          
+        this.setState({
+          wifi_networks: result,
+        });
+      });
+      exec('date')
+      .then(result => {
+        this.setState({ 
+          last_refresh: result
+        })
+        
+      }
+        );
     }, 1000);
   }
 
+
   render() {
     return (
       <div>
-        <Table data={this.state.wifi_networks} />
+        <EnhancedTable rows={this.state.wifi_networks}/>
       </div>
     );
   }
+
 }
 
-function Table(props) {
-  const { data } = props;
-  if (data.length === 0) return <table />;
+
+const headCells = [
+  { id: 'IN-USE', numeric: false, disablePadding: true, label: 'In-Use' },
+  { id: 'BSSID', numeric: false, disablePadding: false, label: 'BSSID' },
+  { id: 'SSID', numeric: false, disablePadding: false, label: 'SSID' },
+  { id: 'MODE', numeric: false, disablePadding: false, label: 'MODE' },
+  { id: 'CHAN', numeric: true, disablePadding: false, label: 'CHAN' },
+  { id: 'RATE', numeric: true, disablePadding: false, label: 'RATE (Mbits/sec)' },
```

Comment:
If the values in the table column already have `Mbits/sec` as suffix, adding it to the table header is redundant.

Please either strip it from the values in the column, or omit it from the table header.

Reply:
Changed in new commit